src/lava/proc/lif/process.py

```diff
@@ -47,12 +47,15 @@ class LIF(AbstractLIF):
               implementations. Ignored for floating point
               implementations.
     vth: Neuron threshold voltage, exceeding which, the neuron will spike.
+    graded_spike: Flag to indicate graded spike. Default is False
```

Comment:
This bool is at a lot of places. See my earlier comment about renaming it to something more verbose like `graded_spike_enabled`